### PR TITLE
Fix mac ci

### PR DESCRIPTION
**Pull request by Jikstra**

### CODE REVIEW

File: ci_scripts/ci-prelude.sh [hunk 1]
```diff
@@ -46,16 +46,28 @@ case $TRAVIS_OS_NAME in
         fi
         ;;
     osx)
-        sudo pip3 install meson
-        ./scripts/build_sasl.sh --with-openssl=/usr/local/opt/openssl
+        # Install meson, ninja, openssl & libetpan
+        brew install openssl meson
+        brew upgrade libetpan
```
Comment (flub) [inline ci_scripts/ci-prelude.sh]: These do not do the same as the `packages` section in `.travis.yml`?  I mean e.g. openssl isn't already brew installed and that the upgrade of libetpan changes anything.  Possibly some travis magic which i misjudged though.

Comment (flub) [reply ci_scripts/ci-prelude.sh]: oh, probably something to do with the brew cleanup bit you added?

Comment (flub) [reply ci_scripts/ci-prelude.sh]: might be worth removing them from travis.yml to avoid confusion about which thing is being used.

Comment (Jikstra) [reply ci_scripts/ci-prelude.sh]: > These do not do the same as the `packages` section in `.travis.yml`? I mean e.g. openssl isn't already brew installed and that the upgrade of libetpan changes anything. Possibly some travis magic which i misjudged though.

Oooh good point. Yes they do. brew cleanup is only for the caching, will modify that.

File: .travis.yml [hunk 2]
```diff
@@ -7,11 +7,11 @@ dist: xenial
 addons:
   homebrew:
     packages:
-      - ninja
+      - meson
```
Comment (flub) [inline .travis.yml]: I assume ninja gets pulled in as a dependency of meson?  I'd be slightly more comfortable with it explicitly listed I think, but no big deal.

Comment (Jikstra) [reply .travis.yml]: yup it's a dependency of meson

**Comment by flub** [comment]
Do you know what the important thing to make this work was here?  Was it the brew cleanup?  The cache?  I just don't understand what changed really.

**Comment by flub** [comment]
Or the `update: true` part?

**Pull request merged**